src/My.Extensions.Localization.Json/JsonStringLocalizer.cs

```diff
@@ -25,7 +25,8 @@ public class JsonStringLocalizer : IStringLocalizer
             string resourceName, // TODO: Use optional parameter in upcoming major release
             ILogger logger)
         {
-            _resourcesPath = resourcesPath ?? throw new ArgumentNullException(nameof(resourcesPath));
+            var path = resourcesPath ?? throw new ArgumentNullException(nameof(resourcesPath));
```

Comment:
Why the below code doesn't work?

Reply:
did not check the reason, just saw the tests are failing and when moving to are they are passing. I could investigate though.

Reply:
@hishamco because it changes the `_searchedLocation` it should change `_resourcesPath` on line [178](https://github.com/shahabganji/My.Extensions.Localization.Json/blob/hotfix/absolutepath/src/My.Extensions.Localization.Json/JsonStringLocalizer.cs#L178)